docs/src/configuration/app/timezone.md

```diff
@@ -10,15 +10,23 @@ That is generally fine but sometimes it's necessary to run cron tasks in a diffe
 
 ## Setting the system timezone for cron tasks
 
-The `timezone` property sets the timezone for which the `spec` property of any [cron tasks](/configuration/app/cron.md) defined by the application will be interpreted.  Its value is one of the [tz database region codes](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) such as `Europe/Paris` or `America/New_York`.  This key will apply to all cron tasks defined in that file.
+The `timezone` property sets the timezone for which the `spec` property of any [cron tasks](/configuration/app/cron.md) defined by the application will be interpreted.
+It defines **relative to which timezone** your cron is being run (to avoid having to calculate to and from UTC).
 
+Its value is one of the [tz database region codes](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) such as `Europe/Paris` or `America/New_York`.  This key will apply to all cron tasks defined in that file.
 This entry is only meaningful on cron specs that specify a particular time of day, rather than a "time past each hour".  For example, `25 1 * * *` would run every day at 1:25 am in the timezone specified.
 
-## Setting an application runtime timezone
 
-The application runtime timezone can also be set, although the mechanism varies a bit by the runtime.
+## Setting an application runtime timezone
 
-* PHP runtime - You can change the timezone by providing a [custom php.ini](/languages/php/ini.md).
+When differing from UTC, the runtime application requires the specific timezone to be set.
+The underlying mechanism on how to set that timezone however depends on the runtime used, for which you can find a non-exhaustive list below:
+* PHP runtime - You can change the timezone by providing a [custom php.ini](/languages/php/ini.md) or by adding the following lines to your `.platform.app.yaml` file:
+```
+variables:
+   php:
+       "date.timezone": "Europe/Paris"
+``` 
```

Comment:
This code block needs to be indented so that it appears as part of the PHP bullet point.  (I hate markdown.)  And a blank line before it on line 25.